scripts/db_migrator.py

```diff
@@ -51,6 +51,34 @@ def migrate_pfc_wd_table(self):
         self.configDB.delete_table('PFC_WD_TABLE')
 
 
+    def migrate_interface_table(self):
+        # Migrate all data entries for Interface table
+        if_db = []
+        if_tables = {
+                     'INTERFACE',
+                     'PORTCHANNEL_INTERFACE',
+                     'VLAN_INTERFACE'
+                    }
+        for table in if_tables:
+            data = self.configDB.get_table(table)
+            for key in data.keys():
+                if not isinstance(key, tuple):
+                    if_db.append(key)
+                    continue
+
+        log_info('Existing interface objects ')
+        log_info(' '.join(if_db))
```

Comment:
there is more like debug information, we only need to log what are the interface migrated.